send.go

```diff
@@ -67,9 +67,10 @@ func (c *Connection) RoundTrip(request *http.Request) (response *http.Response,
 			code = response.StatusCode
 		}
 		labels := map[string]string{
-			metricsMethodLabel: request.Method,
-			metricsPathLabel:   metric,
-			metricsCodeLabel:   strconv.Itoa(code),
+			metricsAPIServiceLabel: c.GetAPIServiceLabelFromPath(request.URL.Path),
```

Comment:
I'm curious, this is different from the way `metricsPathLabel: metric` below obtained the path — that one is taken from X-Metric header.  Will both ways work?

Reply:
it can be done using the same pattern. It only requires to be set at these very top client builders,
https://github.com/openshift-online/ocm-sdk-go/blob/master/connection.go#L603-L620

However, the issue is we need an extra metric param (or hack the existing one) to it and all its descendant client builders. It requires a lot more code updates. Or there could be better way? pls let me know your thought.